src/slices/requests.ts

```diff
@@ -43,7 +43,11 @@ const {
 
 export default requestsSlice.reducer
 
-export const fetchRequests = (): AppThunk => async dispatch => {
+export const fetchRequests = (): AppThunk => async (
+  dispatch,
+  getState,
+  api
```

Comment:
Notice how API is dependency injected here thanks to [_extraArgument_](https://github.com/invalidred/ui-starter-kit/pull/2/files#diff-aefe9ad9c26c1ccaee1145478b95ff38R12) setup. This makes unit testing super easy as no mocking is required.